nginx.conf

```diff
@@ -0,0 +1,78 @@
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+        include       /etc/nginx/mime.types;
+        default_type  application/octet-stream;
+
+        log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+        access_log  /var/log/nginx/access.log  main;
+
+        sendfile        on;
+        #tcp_nopush     on;
+
+        # set client body size to 500M #
+        client_max_body_size 500M;
+
+        keepalive_timeout  65;
+
+        #gzip  on;
+
+        server {
+                listen          80;
+                server_name     localhost;
```

Comment:
this shouldn't just be localhost
![image](https://user-images.githubusercontent.com/4126123/152802285-42c68254-1097-48cb-a905-162ec96deb8d.png)